Scripts/build.swift

```diff
@@ -131,6 +131,35 @@ enum Platform: String, CustomStringConvertible {
         }
     }
 
+    var enableCodeCoverage: Bool {
+        switch self {
+        case .tvOS_10,
+             .tvOS_11,
+             .tvOS_12,
+             .tvOS_13,
+             .macOS_10_12,
+             .macOS_10_13,
+             .macOS_10_14,
+             .macOS_10_15:
+            // We only enable code coverage on our tvOS and macOS targets.
```

Comment:
Why only tvOS and macOS?  Can we do iOS as well?

Reply:
When I started writing this PR, I didn't have a way to run coverage on multiple test targets. Now we do, so enabling tests on all platforms that support it seems reasonable. I'll update!

Reply:
Fixed in https://github.com/square/Valet/pull/202/commits/9e580e544c9c66671692b2e674f6105489315a3a.